src/network.rs

```diff
@@ -161,7 +161,7 @@ impl Network {
         &NETWORKS
     }
 
-    pub fn get(id: &String) -> Option<&'static Network> {
+    pub fn get(id: &str) -> Option<&'static Network> {
```

Comment:
Hmm, I remember doing this and feeling annoyed by it. Something complained about `&str` when I did this. If that seems to be solved, great! :)